itest/cassandra/Dockerfile

```diff
@@ -40,4 +40,7 @@ RUN     chown -R dckruser /nail/cassandra/
 RUN     echo "JVM_OPTS=\"\$JVM_OPTS -Dcassandra.skip_wait_for_gossip_to_settle=0\"" >> /etc/cassandra/cassandra-env.sh
 
 USER    dckruser
+
+# Creates cassandra keyspace and tables
+RUN     cassandra && /opt/setup.sh
```

Comment:
I'm not sure how this is going to work: `cassandra` starts cassandra in the background, then `/opt/setup.sh` creates the tables. On the next line we start cassandra again (in the foreground). When I try this manually I get:

```
java.net.BindException: Address already in use (Bind failed)
        at java.net.PlainSocketImpl.socketBind(Native Method) ~[na:1.8.0_151]
        at java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:387) ~[na:1.8.0_151]
        at java.net.ServerSocket.bind(ServerSocket.java:375) ~[na:1.8.0_151]
        at java.net.ServerSocket.<init>(ServerSocket.java:237) ~[na:1.8.0_151]
        at javax.net.DefaultServerSocketFactory.createServerSocket(ServerSocketFactory.java:231) ~[na:1.8.0_151]
        at org.apache.cassandra.utils.RMIServerSocketFactoryImpl.createServerSocket(RMIServerSocketFactoryImpl.java:33) ~[apache-cassandra-2.2.8.jar:2.2.8]
        at sun.rmi.transport.tcp.TCPEndpoint.newServerSocket(TCPEndpoint.java:666) ~[na:1.8.0_151]
        at sun.rmi.transport.tcp.TCPTransport.listen(TCPTransport.java:330) ~[na:1.8.0_151]
        at sun.rmi.transport.tcp.TCPTransport.exportObject(TCPTransport.java:249) ~[na:1.8.0_151]
        at sun.rmi.transport.tcp.TCPEndpoint.exportObject(TCPEndpoint.java:411) ~[na:1.8.0_151]
        at sun.rmi.transport.LiveRef.exportObject(LiveRef.java:147) ~[na:1.8.0_151]
        at sun.rmi.server.UnicastServerRef.exportObject(UnicastServerRef.java:236) ~[na:1.8.0_151]
        at sun.rmi.registry.RegistryImpl.setup(RegistryImpl.java:213) ~[na:1.8.0_151]
        at sun.rmi.registry.RegistryImpl.<init>(RegistryImpl.java:173) ~[na:1.8.0_151]
        at sun.rmi.registry.RegistryImpl.<init>(RegistryImpl.java:144) ~[na:1.8.0_151]
        at java.rmi.registry.LocateRegistry.createRegistry(LocateRegistry.java:239) ~[na:1.8.0_151]
        at org.apache.cassandra.service.CassandraDaemon.maybeInitJmx(CassandraDaemon.java:119) [apache-cassandra-2.2.8.jar:2.2.8]
        at org.apache.cassandra.service.CassandraDaemon.setup(CassandraDaemon.java:188) [apache-cassandra-2.2.8.jar:2.2.8]
        at org.apache.cassandra.service.CassandraDaemon.activate(CassandraDaemon.java:532) [apache-cassandra-2.2.8.jar:2.2.8]
        at org.apache.cassandra.service.CassandraDaemon.main(CassandraDaemon.java:641) [apache-cassandra-2.2.8.jar:2.2.8]
```

Don't you need to kill the first cassandra process? Or maybe start it on a different port? The itests are clean on your branch so there is definitely something that I'm missing!

Reply:
[cmd](https://docs.docker.com/engine/reference/builder/#cmd) only provides a default command to run when you start a container from this image. It doesn't get executed during the build, so the 2 cassandra processes won't conflict.

Reply:
@drolando awesome, that's what I was missing 👍